### DISCUSSION TITLE
Repo option to download ZIP without parent folder

### DISCUSSION THREAD

**Discussion by InQuognito** [community/community#53332] [Repositories]
### Select Topic Area

Product Feedback

### Body

I have a repository that has a file structure that is very specific, and has to be the way it is due to the way the programs I use compile this file. Whenever I download a ZIP of my code, it will download like this:

`pack.zip/pack/contents`

however, this breaks my very specific file format, and the only fix is to extract the file, go in, find `/contents`, ZIP it, and drag and drop the file into the destination, as opposed to just downloading the ZIP. I imagine there are several others who would benefit from it downloading like this:

`pack.zip/contents`

This would just be a simple settings toggle in the repo that would bring a huge QoL improvement to several developers.

**Comment by Radon8472** [comment]
I have several projects where I have a similar issue, I would need a download without the folder on top, yust the files how the are in my git repo.
I am looking for a solution for a very long time, and I found other with this request too e.g. [Download Github .zip without Repository's name folder]

I would say, there should be an option to controll via `.gitattributes` files, if you like to have this parent folder or not. (Because there are already other options for the zipfile you could controll via a gitignore).
Unfortunally I did not find this option until now, I would be very happy to get any official response to this request.


[Download Github .zip without Repository's name folder]: https://stackoverflow.com/questions/40834928/download-github-zip-without-repositorys-name-folder
